src/scripts/main.js

```diff
@@ -1,7 +1,44 @@
 'use strict';
+// document.createElement('div');
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification');
+
+  switch (type) {
+    case 'success':
+      notification.classList.add('success');
+      break;
+    case 'error':
+      notification.classList.add('error');
+      break;
+    case 'warning':
+      notification.classList.add('warning');
+      break;
+    default:
+      break;
+  }
+
+  notification.getBoundingClientRect();
```

Comment:
The call to `notification.getBoundingClientRect()` is not necessary here as it doesn't affect the notification's display or position. Consider removing it unless you have a specific reason for its inclusion.